types/ember-animated/animated-container.d.ts

```diff
@@ -1,18 +1,21 @@
+import { ElementForTagName } from "@glint/environment-ember-loose/-private/dsl";
 import Component from "@glint/environment-ember-loose/ember-component";
 
-interface AnimatedContainerSignature {
+interface AnimatedContainerSignature<Tag extends string> {
+  /** Multiple tags supported for base element via `tag` arg */
+  Element: ElementForTagName<Tag>;
   Args: {
     motion?: string;
 
     /** Whether to animate the initial render. You will probably also need to set initialInsertion=true on a child component of animated-container. Defaults to false. */
     onInitialRender?: boolean
 
     /** Use a custom tag for the container. Defaults to div. */
-    tag?: String
+    tag?: Tag
   };
   Yields: {
     default: [];
   };
 }
 
-export default class AnimatedContainer extends Component<AnimatedContainerSignature> {}
+declare class AnimatedContainer<Tag extends string = 'div'> extends Component<AnimatedContainerSignature<Tag>> {}
```

Comment:
```suggestion
export default class AnimatedContainer<Tag extends string = 'div'> extends Component<AnimatedContainerSignature<Tag>> {}
```

Reply:
Doh! I am doing it as a declare in my local project I'm using these types in, accidentally copied that over